content/zh/docs/reference/kubernetes-api/authorization-resources/subject-access-review-v1.md

```diff
@@ -0,0 +1,384 @@
+-->
+SubjectAccessReviewSpec 是对访问请求的描述。   
+必须设置 ResourceAuthorizationAttributes 和 NonResourceAuthorizationAttributes 中的一个
+
+<hr>
+
+<!--
+- **extra** (map[string][]string)
+
+  Extra corresponds to the user.Info.GetExtra() method from the authenticator.  Since that is input to the authorizer it needs a reflection here.
+
+- **groups** ([]string)
+
+  Groups is the groups you're testing for.
+
+- **nonResourceAttributes** (NonResourceAttributes)
+
+  NonResourceAttributes describes information for a non-resource access request
+
+  <a name="NonResourceAttributes"></a>
+  *NonResourceAttributes includes the authorization attributes available for non-resource requests to the Authorizer interface*
+
+    - **nonResourceAttributes.path** (string)
+
+    Path is the URL path of the request
+
+  - **nonResourceAttributes.verb** (string)
+
+    Verb is the standard HTTP verb
+-->
+- **extra** (map[string][]string)
+
+  Extra 对应于验证器中的 user.Info.GetExtra()方法。   
+  由于这是授权人的输入，因此需要在此进行显示。
+
+- **groups** ([]string)
+
+  Groups是指你要测试的组。
+
+- **nonResourceAttributes** (NonResourceAttributes)
+
+  NonResourceAttributes 描述非资源请求信息
+
+  <a name="NonResourceAttributes"></a>
+  *NonResourceAttributes 包括可用于向授权者接口发出非资源请求的授权属性*
+
+  - **nonResourceAttributes.path** (string)
+
+    Path 是请求的URL路径
+
+  - **nonResourceAttributes.verb** (string)
+
+    Verb 是标准的 HTTP verb
+
+<!--
+- **resourceAttributes** (ResourceAttributes)
+
+  ResourceAuthorizationAttributes describes information for a resource access request
+
+  <a name="ResourceAttributes"></a>
+  *ResourceAttributes includes the authorization attributes available for resource requests to the Authorizer interface*
+
+  - **resourceAttributes.group** (string)
+
+    Group is the API Group of the Resource.  "*" means all.
+
+  - **resourceAttributes.name** (string)
+
+    Name is the name of the resource being requested for a "get" or deleted for a "delete". "" (empty) means all.
+
+  - **resourceAttributes.namespace** (string)
+
+    Namespace is the namespace of the action being requested.  Currently, there is no distinction between no namespace and all namespaces "" (empty) is defaulted for LocalSubjectAccessReviews "" (empty) is empty for cluster-scoped resources "" (empty) means "all" for namespace scoped resources from a SubjectAccessReview or SelfSubjectAccessReview
+-->
+- **resourceAttributes** (ResourceAttributes)
+
+  ResourceAuthorizationAttributes 描述资源请求信息
+
+  <a name="ResourceAttributes"></a>
+  *ResourceAttributes 包括可用于向授权者接口发出资源请求的授权属性*
+
+  - **resourceAttributes.group** (string)
+
+    Group 是资源的API组。   
+    "*" 表示全部。
+
+  - **resourceAttributes.name** (string)
+
+    Name 是请求 “get” 或删除 “delete” 的资源的名称。   
+    "" (空) 表示全部。
+
+  - **resourceAttributes.namespace** (string)
+
+    Namespace是被请求的操作的命名空间。   
+    目前，没有命名空间和所有命名空间之间没有区别。   
+    LocalSubjectAccessReviews的默认名称空间为“”（空）。   
+    集群范围的资源的默认名称空间为“”（空）。   
+    SubjectAccessReview或SelfSubjectAccessReview的名称空间范围的资源的默认名称空间为“”（空）。
```

Comment:
There should be one space between Mandarin and English word.

Ref: [Localization Guild](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F)